css/site.styl

```diff
@@ -124,16 +132,22 @@ img
 
 // MEDIA QUERIES
 
+grid(96%, s, 18px)
+
 @import "responsive/mobile"
 
 @media screen and (min-width: 720px)
 
+  grid(720px, m, 24px)
+
   @import "responsive/tablet"
 
 // @media screen and (min-width: 1024px)
```

Comment:
should we do a grid for all these common screen sizes? means a little more to check as dev but will make the site more "responsive"